### ISSUE TITLE
Error when using Newsarchive-Menu and news_categories

### ISSUE THREAD

**Issue by laudanju** [codefog/contao-news_categories#92]
Hello Kamil,

I'm using Contao with the news_categories extension, and when I want to implement an archive with the Newsarchive-Menu frontend-module, I get the following error message on all pages containing said frontend-module:

```
Warning: Declaration of NewsCategories\ModuleNewsMenu::compileWeeks($arrData, $strUrl) should be compatible with Contao\ModuleNewsMenu::compileWeeks($arrData) in system/modules/news_categories/modules/ModuleNewsMenu.php on line 21

#0 system/modules/news_categories/modules/ModuleNewsMenu.php(21): __error(2, 'Declaration of ...', '/html/contao-3....', 21, Array)
#1 system/modules/core/library/Contao/ClassLoader.php(161): include('/html/contao-3....')
#2 [internal function]: Contao\ClassLoader::load('ModuleNewsMenu')
#3 [internal function]: spl_autoload_call('ModuleNewsMenu')
#4 system/modules/core/elements/ContentModule.php(43): class_exists('ModuleNewsMenu')
#5 system/modules/core/library/Contao/Controller.php(484): Contao\ContentModule->generate()
#6 system/cache/dca/tl_content.php(1166): Contao\Controller::getContentElement(Object(Contao\ContentModel))
#7 system/modules/core/drivers/DC_Table.php(4321): tl_content->addCteType(Array)
#8 system/modules/core/drivers/DC_Table.php(378): Contao\DC_Table->parentView()
#9 system/modules/core/classes/Backend.php(650): Contao\DC_Table->showAll()
#10 system/modules/core/controllers/BackendMain.php(131): Contao\Backend->getBackendModule('article')
#11 contao/main.php(20): Contao\BackendMain->run()
#12 {main}
```

Is that a known issue or is there a fix for it? The message gets displayed both in front- and backend. 

Contao Version: 3.5.12
news_categories Version: 2.7.2

**Comment by qzminski** [comment]
Can you please try 69d5287?

**Comment by laudanju** [comment]
Seems to work fine, thanks for the help @qzminski ! 👍

**Comment by qzminski** [comment]
Just released 2.7.3.
